Views/Layout.cshtml

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+@inject ICssClassHolder CssClassHolder
+
+<html lang="@Orchard.CultureName()">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><shape type="PageTitle"></shape></title>
+
+    <shape type="Lombiq_BaseTheme_Styles" prop-layoutModel="@Model"></shape>
+
+    <resources type="Header" />
+</head>
+<body dir="@Orchard.CultureDir()" class="@string.Join(" ", CssClassHolder.Body)">
```

Comment:
This join could be an extension method, too. Could be added to the Helpful Libraries.